src/br/com/zup/nossacasacodigo/auxiliars/ShowABook.java

```diff
@@ -0,0 +1,42 @@
+package br.com.zup.nossacasacodigo.auxiliars;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.util.Optional;
+import br.com.zup.nossacasacodigo.author.Author;
+import br.com.zup.nossacasacodigo.book.Book;
+import br.com.zup.nossacasacodigo.category.Category;
+import br.com.zup.nossacasacodigo.database.Database;
+
+public class ShowABook {
+
+	public static void main(String[] args) {
+		String title = "Spring Boot";
+		String synopsis = "Descri��o do livro";
+		String summary = "Acelere o desenvolvimento de microsservi�os";
+		BigDecimal price = new BigDecimal("29.9");
+		int pages = 154;
+		String isbn = "978-85-94120-00-7";
+		LocalDate publicationDate = LocalDate.of(2020,8,16);
+		Category category = new Category("cl�ssico moderno");
+		
+		String name = "Fernando Boaglio";
+		String email = "larissafabiao@gmail.com";
+		String description = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin et placerat purus. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Pellentesque vulputate libero tortor. Fusce arcu felis, rhoncus eget est in, suscipit venenatis orci. Vestibulum eget feugiat justo. Morbi sit amet felis finibus, imperdiet metus eget, pellentesque risus. Nulla at porttitor turpis.";
+		
+		Author fernandoBoaglio = new Author(name, email, description);
+		
+		Book springBoot = new Book(title, synopsis, summary, price, pages, isbn, category, publicationDate, fernandoBoaglio);
+		Database database = new Database();
+		database.addBookInDatabase(springBoot);
+		
+		Optional<Book> bookReturned = database.searchBook("Spring Boot");
+		
+		if (bookReturned.isPresent()) {
+			Book book = bookReturned.get();
+			book.printBookInfo();
```

Comment:
Esse daqui é um ótimo ponto de reflexão, vi que @yurioliveirazup começou a falar sobre o assunto em outro comentário também e que vocês já começaram a debater. Não existe problema em ter y ou x métodos se eles realmente são necessários :). 

Por exemplo, neste caso aqui: como que seu método printBookInfo sabe a formatação do valor? Como ele sabe que precisa printar "Conteudo:" antes do conteúdo em si?  

Muitas vezes, quando precisamos escrever o código que chama e também o código que implementa, criamos um acoplamento mental que nem sentimos. Se o preço para não estar acoplado for alto, provavelmente vale a pena começar. Só que neste caso ficar acoplado ou não aumenta(ou aumenta pouco) o nível de dificuldade do entendimento do código(dada a régua que estamos trabalhando). 

@yurioliveirazup, não sei se já explicou, mas acho que chegou o momento de explicar para Larissa e Maurício a nossa métrica de dificuldade de entendimento :).